### PR TITLE
Strip trailing newline from timestamp

### PR DESCRIPTION
**Pull request by nhibberd**
@markhibberd  @jystic @charleso

### CODE REVIEW

File: src/Setup.hs [hunk 1]
```diff
@@ -66,4 +66,4 @@ gitVersion = do
 
 timestamp :: IO String
 timestamp =
-  readProcess "date" ["+%Y%m%d%H%M%S"] ""
+  fmap init $ readProcess "date" ["+%Y%m%d%H%M%S"] ""
```
Comment (jystic) [inline src/Setup.hs]: Savage

Comment (jystic) [inline src/Setup.hs]:
```
import Data.Char (isDigit)
-- snip
fmap (takeWhile isDigit) $ readProcess "date" ["+%Y%m%d%H%M%S"] ""
```

Comment (jystic) [inline src/Setup.hs]: or maybe:
```haskell
import Data.Char (isSpace)
-- snip
reverse . dropWhile isSpace . reverse
```

**Comment by nhibberd** [comment]
@jystic bump

**Comment by jystic** [comment]
🌕🌖🌗🌘🌑🌒🌓🌔🌕

**Pull request merged**